miprometheus/problems/seq_to_seq/seq_to_seq_problem.py

```diff
@@ -75,6 +77,86 @@ def evaluate_loss(self, data_dict, logits):
         return loss
 
 
+    def to_dictionary_indexes(self, dictionary, sentence):
```

Comment:
All those methods look for me as related to text. SeqToSeq problems abstract from text.

I guess it is good to move them to a class higher in the hierarchy (from bAbI), as probably they will be reused by the other multi-question version.

I propose to introduce a new class, inheriting from SeqToSeq, that will contain those methods, and maybe other methods shared with the second version of bAbI...

Reply:
Do you think those methods can be useful in e.g. machine translation?

Reply:
These should be moved to `TextToText` or another class related to NLP

Reply:
yes ok let's go for TextToText